core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/impl/columns/Utils.kt

```diff
@@ -189,8 +189,9 @@ internal fun <A, B> SingleColumn<A>.transformSingle(
  * or it can be used as a [TransformableColumnSet]<[B]>, where a [ColumnsResolverTransformer] can be injected before
  * the [converter] is applied.
  */
-internal fun <A, B> ColumnsResolver<A>.transform(
-    converter: (List<ColumnWithPath<A>>) -> List<ColumnWithPath<B>>,
+@PublishedApi
+internal inline fun <A, B> ColumnsResolver<A>.transform(
+    crossinline converter: (List<ColumnWithPath<A>>) -> List<ColumnWithPath<B>>,
 ): TransformableColumnSet<B> =
     object : TransformableColumnSet<B> {
```

Comment:
remember me someday to remove this TransformableColumnSet stuff. It was the result of `.recursively()`. I have a hunch we can inline more if `transformResolve` doesn't exist anymore :)